sdk/src/wallet/migration/mod.rs

```diff
@@ -1,6 +1,8 @@
 // Copyright 2023 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
+#[cfg(not(target_family = "wasm"))]
+pub(crate) mod chrysalis;
```

Comment:
Nit but I'd make this private and export `key_to_chrysalis_key` if needed, feel free to ignore

Reply:
Would rather keep like it is, it's also `key_to_chrysalis_key` and reexporting them separated would be more code

Reply:
Fair enough